software/librephotos.yml

```
@@ -12,3 +12,4 @@ source_code_url: https://github.com/LibrePhotos/librephotos
 stargazers_count: 6135
 updated_at: '2023-11-19'
 archived: false
+related_software_url: https://uhuru.photos/
```

```suggestion
related_software_url: https://docs.librephotos.com/docs/user-guide/mobile/
```

There is another, official mobile client. The linked page also lists UhuruPhotos.